src/ai/backend/accelerator/cuda_open/plugin.py

```diff
@@ -67,7 +67,8 @@
 
 PREFIX = "cuda"
 
-log = BraceStyleAdapter(logging.getLogger("ai.backend.accelerator.cuda"))
+logger = logging.getLogger("ai.backend.accelerator.cuda")
+log = BraceStyleAdapter(logger)
```

Comment:
Hm... Could we revert back these changes to the original to avoid exposing additional unused symbol in the global module namespace (e.g., when auto-completing in IDEs)?

Reply:
Yes! This was from the previous way of handling graylog, it doesn't have to be like this now. Thank you for noticing it